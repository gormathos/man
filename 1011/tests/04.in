sol_id;user_id;task_id;score
161020;1004;2005;13.64
161021;1004;2003;16.33
161021;1003;2008;58.14
161024;1001;2008;27.91
161034;1001;2001;95.00
161043;1004;2007;28.00
161046;1001;2002;52.50
161046;1001;2008;2.33
161046;1001;2006;97.62
161047;1002;2008;83.72
161057;1003;2003;22.45
161063;1002;2001;60.00
161067;1004;2006;9.52
161077;1001;2001;35.00
161077;1002;2005;18.18
161085;1004;2005;0.00
161090;1004;2001;0.00
161097;1001;2002;10.00
161106;1003;2001;10.00
161106;1001;2003;6.12
161110;1002;2007;40.00
161115;1002;2001;30.00
161115;1004;2008;34.88
161120;1004;2001;95.00
161127;1003;2005;54.55
161133;1001;2004;23.91
161139;1003;2006;16.67
161143;1003;2006;19.05
161146;1002;2003;40.82
161153;1001;2006;40.48
161156;1001;2007;8.00
161163;1004;2002;77.50
161164;1004;2006;45.24
161173;1002;2003;8.16
161178;1002;2008;4.65
161184;1004;2003;4.08
161184;1004;2004;34.78
161186;1003;2007;52.00
161196;1004;2003;73.47
161201;1004;2002;35.00
161206;1004;2005;45.45
161212;1004;2007;20.00
161214;1001;2005;0.00
161220;1001;2003;24.49
161226;1001;2001;65.00
161227;1004;2007;20.00
161230;1003;2005;27.27
161238;1002;2003;69.39
161247;1002;2002;50.00
161250;1004;2004;4.35
161257;1004;2001;0.00
161259;1003;2008;60.47
161259;1003;2008;11.63
161263;1003;2004;45.65
161269;1004;2002;22.50
161279;1004;2007;40.00
161284;1002;2007;0.00
161285;1003;2007;36.00
161285;1002;2004;69.57
161288;1002;2004;28.26
161295;1001;2008;6.98
161301;1001;2001;40.00
161308;1003;2007;44.00
161311;1004;2008;6.98
161314;1003;2006;7.14
161321;1003;2003;2.04
161330;1003;2006;76.19
161332;1003;2004;39.13
161338;1004;2003;12.24
161342;1002;2006;35.71
161346;1001;2005;4.55
161347;1001;2002;10.00
161348;1004;2005;63.64
161350;1003;2002;10.00
161356;1001;2002;15.00
161362;1002;2006;76.19
161366;1004;2008;34.88
161368;1001;2007;36.00
161372;1002;2002;52.50
161376;1004;2004;6.52
161384;1002;2003;69.39
161391;1004;2006;57.14
161401;1003;2001;50.00
161408;1003;2003;0.00
161411;1001;2007;16.00
161419;1004;2003;30.61
161419;1001;2004;34.78
161425;1002;2002;27.50
161425;1003;2007;40.00
161431;1002;2004;82.61
161440;1003;2004;13.04
161448;1002;2002;60.00
161458;1003;2005;9.09
161466;1002;2008;34.88
161468;1004;2004;17.39
161472;1003;2002;35.00
161480;1003;2008;23.26
161481;1004;2007;20.00
161487;1002;2001;95.00
161488;1003;2004;26.09
161496;1002;2007;8.00
161501;1001;2005;54.55
161511;1003;2001;5.00
161516;1001;2001;30.00
161518;1002;2005;9.09
161524;1001;2004;17.39
161533;1002;2003;100.00
161535;1004;2008;9.30
161541;1002;2008;79.07
161548;1001;2006;54.76
161550;1004;2005;95.45
161555;1003;2002;15.00
161555;1002;2007;0.00
161563;1001;2006;16.67
161567;1003;2002;2.50
161569;1001;2003;18.37
161574;1003;2006;0.00
161579;1002;2006;21.43
161582;1002;2002;45.00
161585;1002;2005;95.45
161594;1002;2006;14.29
161598;1002;2001;35.00
161601;1002;2008;41.86
161609;1003;2004;2.17
161618;1003;2005;81.82
161623;1004;2008;0.00
161624;1001;2008;74.42
161633;1001;2007;20.00
161638;1001;2005;36.36
161638;1001;2006;66.67
161643;1002;2004;10.87
161649;1003;2001;20.00
161652;1002;2005;9.09
161662;1001;2005;4.55
161670;1002;2006;61.90
161674;1004;2002;27.50
161682;1001;2007;0.00
161686;1004;2001;50.00
161693;1001;2004;54.35
161693;1003;2005;50.00
161698;1003;2008;2.33
161702;1001;2003;36.73
161710;1002;2001;10.00
161711;1004;2004;0.00
161716;1001;2003;2.04
161717;1003;2003;61.22
161725;1003;2003;18.37
161726;1002;2004;82.61
161729;1001;2004;32.61
161730;1004;2006;16.67
161732;1003;2001;75.00
161736;1002;2005;27.27
161743;1003;2002;67.50
161745;1002;2007;16.00
161753;1001;2002;52.50
161760;1003;2007;20.00
161764;1004;2001;30.00
161768;1004;2002;7.50
161772;1001;2008;6.98
161780;1004;2006;28.57
