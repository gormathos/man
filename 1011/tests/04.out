1001: Very Good [62.65%]
1002: Excellent [79.12%]
1003: Very Good [64.98%]
1004: Very Good [63.53%]
